packages/odyssey-react-mui/src/labs/Layout.tsx

```diff
@@ -76,14 +67,14 @@ const Layout = ({ regions, children }: LayoutProps) => {
     .join(" ");
 
   return (
-    <LayoutContainer odysseyDesignTokens={odysseyDesignTokens}>
+    <Box>
```

Comment:
@jordankoschei-okta What are we looking to do here by removing this spacing?

Reply:
It was entirely redundant — the spacing is already there, thanks to `gap`, and this was doubling it up so the vertical spacing didn't match the horizontal spacing.